tools/tpm2_dump_capability.c

```diff
@@ -600,7 +600,7 @@ dump_tpm_capability (TPMU_CAPABILITIES    *capabilities,
     case TPM_CAP_COMMANDS:
         dump_command_attr_array (capabilities->command.commandAttributes,
                                  capabilities->command.count);
-        /* fallthrough */
+        break;
```

Comment:
Yes, according to its documentation dump_tpm_capability should return 1 only when it fails to handle the parameters. But the only caller of this function ignores the return value, so nothing was actually broken.